modules/sfp_massdns.py

```diff
@@ -0,0 +1,406 @@
+# -*- coding: utf-8 -*-
```

Comment:
Modules which leverage third-party tools on the host use a `sfp_tool_` prefix for the module name and file name (`sfp_tool_massdns.py` rather then `sfp_massdns.py`).